src/main/java/io/cdap/plugin/snowflake/source/batch/SnowflakeBatchSourceConfig.java

```diff
@@ -87,6 +109,15 @@ public String getSchema() {
     return schema;
   }
 
+  @Nullable
+  public String getTableName() {
+    return tableName;
+  }
+
+  public ImportQueryType getImportQueryType() {
```

Comment:
this can be nullable

Reply:
Added the annotation `@Nullable` to this method. f1e33c02ce1b80b5471c0770c455fd65b3246e1a